### PR TITLE
Updated transaction icons

### PR DESCRIPTION
**Pull request by captainbuckkets**
Updated to be a little spicier

### CODE REVIEW

**Comment by bananenwilly** [comment]
I guess the address book could use the same icons then?

**Comment by captainbuckkets** [comment]
Yeah I could.  Still going through everything and trying to make responsible :P

**Comment by captainbuckkets** [comment]
Having to relearn a bunch so going slow

**Comment by bananenwilly** [comment]
regarding your changes in the about tab:
 style: TextButton.styleFrom(
                                  primary: MyTheme.peercoinGreen,
                                ),

does look exactly the same to me as the current implementation. 
Screenshot_1621235661
(first button has your change, rest has not been changed)

Also, please use Theme.of(context).primaryColor instead of hard linking peercoinGreen

**Comment by captainbuckkets** [comment]
Okay can do.  Was confused on the theming linking stuff.  BTW it forces to system default with no option to go back to light mode

**Comment by captainbuckkets** [comment]
Its weird that it won't show it changed for the first entry

**Comment by bananenwilly** [comment]
Ah you mean when the system is on dark mode you want a local option that would overwrite it to light mode or vice versa?

File: lib/tools/app_themes.dart [hunk 1]
```diff
@@ -32,15 +32,20 @@ class MyTheme {
...
-        primaryColor: Color(0xff46662B),
-        scaffoldBackgroundColor: DarkThemeColors.background,
-        accentColor: DarkThemeColors.primaryAccent,
-        errorColor: Color(0xffAB0C3D),
-        elevatedButtonTheme: ElevatedButtonThemeData(
-            style: ElevatedButton.styleFrom(
-          primary: Color(0xff46662B),
-          onPrimary: Colors.white,
-        )))
+      primaryColor: peercoinGreen,
```
Comment (bananenwilly) [inline lib/tools/app_themes.dart:35]: I think that makes the title / app bar too bright in dark mode.. we should stick with the old color

**Review by bananenwilly — APPROVED** [review]

**Pull request merged**